packages/yoga/src/Button/web/Link.jsx

```diff
@@ -68,9 +68,14 @@ const Link = styled(Button)`
   }}
 `;
 
-const ButtonLink = ({ disabled, ...props }) => (
-  <Link {...props} disabled={disabled} aria-disabled={disabled} />
-);
+// eslint-disable-next-line react/prop-types
+const ButtonLink = forwardRef(({ isLoading, ...rest }, ref) => {
```

Comment:
Where are we using this destructured `isLoading`?

Reply:
Do you think is scalable if you get all of the other props except the `isLoading`? This means every time someone adds a new value for all buttons they will need to add it here too, otherwise it will be ignored. WDYT?

Reply:
Good point 🤔

I'm just concerned that someone might misunderstand the 'button.text' and 'button.link' props and try to use them. By removing this approach, people will still be able to use isLoading, and it will work. However, it wouldn't be advisable to do so because, as far as I know, we don't have any plans for that.

But, yes I agree with you. This approach isn't scalable. It's more on the developer's side to use it correctly.

Reply:
No no, I really got your point and I think you're right, I'm just wondering if we can handle it in another way instead of having this eslint comment

Reply:
What if you do something like this? 

```JSX
const ButtonLink = forwardRef(({ rest }, ref) => {

const props = Object.fromEntries(
  Object.entries(rest).filter(([key]) => key !== 'isLoading')
);
  
<Link {...props} disabled={props.disabled} aria-disabled={props.disabled} ref={ref} /> 
```

Reply:
Great suggestion!! 🚀